src/main/groovy/com/github/jrubygradle/internal/GemVersionResolver.groovy

```diff
@@ -4,10 +4,52 @@ import org.gradle.api.artifacts.DependencyResolveDetails
 
 class GemVersionResolver {
 
-    private Map versions = [:]
-  
+    Map versions
+    private final Object configuration
+    private final Object logger
+
+    static void setup(Object project) {
+        project.configurations.each {
```

Comment:
that is true but how do I recognize a configuration with gems ? look at all its top level dependencies including the inherited configurations ? any other idea is OK as well